spec/unit/queue_storage_spec.rb

```diff
@@ -52,8 +45,12 @@ module Backend
       private
 
       def is_sentinel?(connection)
-        connector = connection.instance_variable_get(:@inner)
-                              .instance_variable_get(:@client)
+        if ThreeScale::Backend.configuration.redis.async
+          connector = connection.instance_variable_get(:@redis_async)
+          return connector.instance_of?(Async::Redis::SentinelsClient)
+        end
+
+        connector = connection.instance_variable_get(:@client)
```

Comment:
if/else would read better for me here as well

Reply:
OK